File: microservicio/dominio/src/main/java/com/ceiba/seats/modelo/entidad/Seat.java
package com.ceiba.seats.modelo.entidad;

import lombok.Getter;

@Getter
public class Seat {


    Integer idSeat;
    String numberSeat;
    Integer available;
    Integer idTicket;


    public Seat(Integer idSeat,String numberSeat,Integer available, Integer idTicket){

        this.idSeat = idSeat;
        this.numberSeat = numberSeat;
        this.available = available;
        this.idTicket = idTicket;


    }
}

File: angular-base/src/app/feature/ticket/shared/model/seat.ts
export class Seat{
    id:number;
    number_seat:string;
    available:number;
    id_ticket:number;
    id_movie_projector:number;

    constructor(id:number,number_seat:string,available:number,id_ticket:number,id_movie_projector:number){
        this.id = id;
        this.number_seat = number_seat;
        this.available = available;
        this.id_ticket = id_ticket;
        this.id_movie_projector=id_movie_projector;
    }

}